Ships as Raw

```diff
@@ -0,0 +1,860 @@
+
+Name|Gladiator I-class Star Destroyer
+Type|Gladiator
+Keywords|0
+Faction|Imperial
+Size|Small
+Points|56
+Hull|5
+defenseTokens|Br Rd Ev
+Command|2 
+Squad|2 
+Eng|3 
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|3
+SideShields|2
+RearShields|1
+FrontArmament|RRKK
+SideArmament|KKKK
+RearArmament|RK
+AntiSquadron|B
+Upgrade|Title Officers Weapon Team Support Ordance

```

Comment:
Remove the space in WeaponTeam, or remove Team.  Basically lets have as many words as upgrades.  I think you have an extra word here.